### PR TITLE
Adds a more specific error message and passes the exception,

### PR DESCRIPTION
**Pull request by shoppel**
otherwise the process cannot be stopped anymore except by killing
(happened in camera_usb when converting UYVY).

### CODE REVIEW

File: tasks/Task.cpp [hunk 1]
```diff
@@ -144,10 +145,13 @@ void Task::processImage() {
     catch(std::runtime_error e)
     {
         RTT::log(RTT::Error) << "processing error: " << e.what() << RTT::endlog();
-        RTT::log(RTT::Error) << "Have you specified camera_format and output_format right?" << RTT::endlog();
         report(PROCESSING_ERROR);
+        err = true;
     }
     output_frame.reset(frame_ptr);
+    if(err) {
+        throw std::runtime_error("Conversion is not available, try another camera_format and output_format");
```
Comment (goldhoorn) [inline tasks/Task.cpp:153]: Throw is really hard for such kind of "expectable error" i suggest it would be better to switch to error or exception state.

**Comment by goldhoorn** [comment]
@shoppel : pong

**Comment by goldhoorn** [comment]
Btw. you have to reopen this PR, you done the PR against the rc-branch which get never merged into anything. please reopen against master if you still want this change.

**Comment by shoppel** [comment]
Will reopen this pull request on master and using exception states.

**Pull request closed without merging**